lib/tlaw/dsl.rb

```diff
@@ -333,19 +362,19 @@ module DSL
     #     Usage:
     #
     #     ```ruby
-    #     post_process('date') { |val| Date.parse(val) }
+    #     transform('date') { |val| Date.parse(val) }
     #     # or, btw, just
-    #     post_process('date', &Date.method(:parse))
+    #     transform('date', &Date.method(:parse))
     #     ```
     #
     #     @param key [String]
 
-    # @!method post_process_items(key, &block)
+    # @!method transform_items(key, &block)
```

Comment:
Let's make it `transform_each`, WDYT? I don't know where was my head when I called that `_items` :( I was younger, though.

Reply:
Haha. My opinion is that the transformations should return the new value, not mutate it. So, I would probably want it to be `map` maybe. I meant to note this if I didn't already. This goes for all transformations, not just this one.